examples/node_pool/variables.tf

```diff
@@ -77,3 +77,11 @@ variable "cluster_autoscaling" {
   }
   description = "Cluster autoscaling configuration. See [more details](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#clusterautoscaling)"
 }
+
+variable "network_config" {
```

Comment:
Don't make this a variable. Just hard code it. Examples can *and should* use hardcoding extensively.

Reply:
np, it just will make an assumption on users env. will change.

Reply:
fixed